include/wx/generic/listctrl.h

```diff
@@ -130,6 +130,7 @@ class WXDLLIMPEXP_CORE wxGenericListCtrl: public wxNavigationEnabled<wxListCtrlB
     bool EndEditLabel(bool cancel);
 
     wxTextCtrl* GetEditControl() const;
+    bool IsVisible(long item);
```

Comment:
```suggestion
    bool IsVisible(long item) wxOVERRIDE;
```
Function is not really needed, as the implementation directly calls the base version.

Reply:
Agreed.
Will be removed.